index.html

```diff
@@ -0,0 +1,1606 @@
+              target="_"
+            >
+              <img src="./svgs/nav-twitter.svg" alt="twitter_icon" />
+            </a>
+          </div>
+        </div>
+        <div class="navbar__title">
+          <img src="./svgs/zen-brush-symbol.svg" alt="zen_icon" />
+          <span class="navbar__title__text">Satori UI</span>
+        </div>
+      </nav>
+      <!-- end of the navbar -->
+
+      <!-- start of the container -->
+      <div class="container">
+        <!-- start of the sidebar-->
+        <div class="sidebar">
+          <ul class="sidebar__list">
+            <span class="sidebar__title">COMPONENTS</span>
+            <a href="#avatar" class="sidebar__item">Avatar</a>
+            <a href="#alerts" class="sidebar__item">Alert</a>
+            <a href="#badge" class="sidebar__item">Badge</a>
+            <a href="#buttons" class="sidebar__item">Buttons</a>
+            <a href="#cards" class="sidebar__item">Cards</a>
+            <a href="#image" class="sidebar__item">Image</a>
+            <a href="#inputs" class="sidebar__item">Input</a>
+            <a href="#lists" class="sidebar__item">Lists</a>
+            <a href="#navigation" class="sidebar__item">Navigation</a>
+            <a href="#tomodal" class="sidebar__item">Modal</a>
+            <a href="#rating" class="sidebar__item">Rating</a>
+            <a href="#slider" class="sidebar__item">Slider</a>
+            <a href="#two-col-grid" class="sidebar__item">Simplified Grid</a>
+            <a href="#text-utilities" class="sidebar__item">Text Utilities</a>
+            <a href="#snackbar" class="sidebar__item">Snackbar</a>
+          </ul>
+        </div>
+        <!-- end of the sidebar -->
+
+        <!-- start of the component container -->
+        <div class="component-container center--margin">
+          <h1 id="avatar" class="component__avatar component--mod">Avatar</h1>
+
+          <!-- start of the avatar container -->
+          <div class="avatar space-comp">
+            <div class="avatar-display">
+              <img
+                class="img-big img-utils"
+                alt="avatar-small"
+                src="../resources/man.svg"
+                width="96"
+                height="96"
+              />
+              <img
+                class="img-medium img-utils"
+                alt="avatar-medium"
+                src="../resources/man.svg"
+                width="64"
+                height="64"
+              />
+              <img
+                class="img-small img-utils"
+                alt="avatar-medium"
+                src="../resources/man.svg"
+                width="48"
+                height="48"
+              />
+              <img
+                class="img-ex-small img-utils"
+                alt="avatar-extra-small"
+                src="../resources/man.svg"
+                width="32"
+                height="32"
+              />
+            </div>
+            <pre>
```

Comment:
prism code is having a scrollbar on the right side, unnecessary

Reply:
Fixed.